ontology/yaml/resources/SAFETY/entity_types/GENERALTYPES.yaml

```diff
@@ -103,4 +103,9 @@ SSS:
   is_abstract: true
   implements:
   - /SENSOR
-  
\ No newline at end of file
+
+SENSOR:
```

Comment:
You do not need to define this here. The `/SENSOR` in the other yaml should suffice.

Reply:
Ohkay Thanks. Done the required changes.

Reply:
Please remove the SENSOR type from this file.